packages/genesys-spark-components/src/components/beta/gux-tooltip-beta/gux-tooltip.scss

```diff
@@ -0,0 +1,7 @@
+:host {
+  display: none;
+}
+
+:host(.gux-show) {
+  display: block;
+}
```

Comment:
Mimic the display state of the base-tooltip in light dom elements so there's no tooltip role without shadow dom tooltip content.